src/main/groovy/life/qbic/datamodel/dtos/portal/PortalUser.groovy

```diff
@@ -0,0 +1,46 @@
+package life.qbic.datamodel.dtos.portal
+
+import groovy.transform.Immutable
+
+/**
+ * <p>A user within QBiC's data management platform.</p>
+ *
+ * <p>For example a user with the full name <strong>Max Maxwell Minestrone</strong> has
+ * the first name <i>Max</i>, the middle name <i>Maxwell</i> and the last name <i>Minestrone</i>.</p>
+ *
+ * @since 2.9.0
+ */
+@Immutable
+class PortalUser {
+
+    /**
+     * The portal system's user id
+     * @since 2.9.0
+     */
+    String id
+
+    /**
+     * The full name of the user
+     * @since 2.9.0
+     */
+    String fullName
```

Comment:
This is kind of redundant information since you can retrieve that information from the first (, middle) and last name

Reply:
true, i wanted to provide a getter for the full name, I will provide one